site/docs/guides/contributor/release-process/egeria-core-release.md

```diff
@@ -0,0 +1,58 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the Egeria project 2020. -->
+
+### Egeria Core Release process
+
+#### Obtaining releases / artifacts for Core Egeria
+| Location | Usage |
+|---|---|
+| [Maven Central core Egeria release :material-dock-window:](https://mvnrepository.com/artifact/org.odpi.egeria){ target=maven } | typically used by other developers integrating with our code |
+| [Github core Egeria Release :material-github:](https://github.com/odpi/egeria/releases){ target=gh } | source code in `zip` and `tar.gz` formats |
+| `git` | `git checkout Vx.y` to get version as-shipped (each release is tagged at the point it is shipped) |
+
+### Core Egeria Release process
+
+??? success "1. Agree schedule"
+    --8<-- "docs/guides/contributor/release-steps/agree-schedule.md"
+??? success "2. Track remaining issues and PRs"
+    --8<-- "docs/guides/contributor/release-steps/track-outstanding.md"
+??? success "3. Create branch"
+    - Checkout master `git checkout master`
+    - Ensure local update `git pull upstream master`
```

Comment:
after this I check my master REALLY IS the same as upstream with
`git reset --hard upstream/master` (noting that this could override any local changes)